lib/api/api_service.dart

```diff
@@ -10,7 +10,7 @@ class ApiService {
 
   late final ApiClient _apiClient = ApiClient();
 
-  late PokemonApi? _pokemonApi;
+  PokemonApi? _pokemonApi;
```

Comment:
any reason why `late` keyword was removed? 🤔 

late keyword was used for that object to be lazily initialized - meaning you don't need to initialize on the spot rather you can have the option when and where to... 🙂

Reply:
I have encountered an error `Unhandled Exception: LateInitializationError: Field '_pokemonApi@27342008' has not been initialized.` and with this I just initialized _pokemonApi at first and did not make it a lazy variable